etc/base.yaml.gotmpl

```diff
@@ -10,3 +10,8 @@ radar_grafana:
     allprojects:
       home:
         json: {{ readFile "radar-grafana/dashboards/allprojects/home.json" | quote }}
+
+radar_appserver:
+  google_application_credentials: {{ readFile "../etc/radar-appserver/google-credentials.json" | quote }}
```

Comment:
maybe could do with some instructions regarding this process in the README?

Reply:
Should it be added in the main README?

Reply:
Ok I've added it here in the meantime https://github.com/RADAR-base/RADAR-Kubernetes/blob/add-remaining-charts/README.md#optional